app_simple.py

```
@@ -6,11 +6,45 @@
 import base64
 import io
 from PIL import Image, ImageDraw
+import openai
```

Ruff (0.8.2)

`openai` imported but unused

Remove unused import: `openai`

(F401)